toffy/tiling_utils.py

```diff
@@ -502,22 +503,25 @@ def generate_tiled_region_fov_list(tiling_params, moly_path):
     Args:
         tiling_params (dict):
             The tiling parameters created by `set_tiled_region_params`
-        moly_path (str):
+        moly_path (Optional[str]):
             The path to the Moly point to insert between FOV intervals and/or regions.
             If these insertion parameters are not specified in `tiling_params`, this won't be used.
+            Defaults to None.
 
     Returns:
         dict:
             Data containing information about each FOV
     """
 
     # file path validation
-    if not os.path.exists(moly_path):
-        raise FileNotFoundError("Moly point file %s does not exist" % moly_path)
+    if (tiling_params.get("moly_region", "N") == "Y") or \
+       (tiling_params.get("moly_interval", 0) > 0):
+        if not os.path.exists(moly_path):
+            raise FileNotFoundError("Moly point file %s does not exist" % moly_path)
```

Comment:
```suggestion
            raise FileNotFoundError("The provided Moly FOV file %s does not exist. If you want to include Moly FOVs you must provide a valid path. Otherwise, select 'No' for the options relating to Moly FOVs" % moly_path)
```